src/orquestra/sdk/_base/_workflow.py

```diff
@@ -523,6 +544,8 @@ def workflow(
     *,
     data_aggregation: Optional[Union[DataAggregation, bool]] = None,
     custom_name: Optional[str] = None,
+    default_source_import: Optional[Import] = None,
+    default_dependency_imports: Optional[Iterable[Import]] = None,
```

Comment:
**issue**: please describe in the docstring (or on the docs( what's the implication of using vs not using `default_source_import` and `default_dependency_imports`. The `@sdk.workflow` decorator is the main component of this library, from all symbols we export this one should have the best documentation.